source/brailleDisplayDrivers/hims.py

```diff
@@ -237,9 +282,12 @@ def __init__(self, port="auto"):
 		for match in self._getTryPorts(port):
 			portType, portId, port, portInfo = match
 			self.isBulk = portType == bdDetect.DeviceType.CUSTOM
+			self.isHID = portType == bdDetect.DeviceType.HID
 			# Try talking to the display.
 			try:
-				if self.isBulk:
+				if self.isHID:

```

Comment:
Please consider using a match statement, like this

```suggestion
				match portType:
					case bdDetect.DeviceType.HID:
```

Reply:
Thanks. I converted it with match case.